packages/cli/src/cms/api/types.ts

```diff
@@ -48,33 +48,42 @@ export type Remote = {
 export type PushResponse = {
   id: string;
   remoteId: string;
+  isMainBranch: boolean;
+  isOutdated: boolean;
+  hasChanges: boolean;
+  replace: boolean;
+  scoutJobId: string | null;
+  uploadedFiles: Array<{ path: string; mimeType: string }>;
   commit: {
-    message: string;
     branchName: string;
-    sha: string | null;
-    url: string | null;
+    message: string;
     createdAt: string | null;
-    namespace: string | null;
-    repository: string | null;
+    namespaceId: string | null;
+    repositoryId: string | null;
+    url: string | null;
+    sha: string | null;
     author: {
       name: string;
       email: string;
       image: string | null;
     };
+    statuses: Array<{
+      name: string;
+      description: string;
+      status: 'pending' | 'running' | 'success' | 'failed';
+      url: string;
```

Comment:
```suggestion
      url: string | null;
```

Reply:
When there are no changes or files are outdated we don't start build and don't have URL

Reply:
Thanks, updated